llm_demo/orchestrator/vertexai_function_calling/functions.py

```diff
@@ -22,7 +22,11 @@
 
 search_airports_func = generative_models.FunctionDeclaration(
     name="airports_search",
-    description="Use this tool to list all airports matching search criteria. Takes at least one of country, city, name, or all of the above criteria. This function could also be used to search for airport information such as iata code.",
+    description="""
```

Comment:
In the future, we might want to deduplicate these instructions across the orchestration modules

Reply:
Yes, agree~ We can definitely dedup the tools / functions. I'll add this to the backlog :)